src/test/java/uk/gov/digital/ho/pttg/AuditEntryJpaRepositoryTest.java

```diff
@@ -181,11 +182,60 @@ public void findAuditHistory_filtersByEventType() {
         assertThat(all).size().isEqualTo(0);
     }
 
+    /**
+     * As this test involves some Postgres specific syntax for the JSONB column, the intention was to run this test
+     * against an in memory postgres database.  However, I was unable to get such a database working consistently in
+     * conjunction with Spring Boot.
+     *
+     * This test was run manually against postgres 9.6 running in a docker container, and it passed there, so we do at
+     * least know the test passes against a real postgres db.
+     *
+     * To run this test:
+     * - add the class level annotation @ActiveProfiles("postgres,postgresDocker")
+     * - run a docker container with the following command (replacing with the correct version of postgres):
+     *     docker run  --name pg-docker -e POSTGRES_PASSWORD=docker -d -p 5432:5432 postgres:9.6.12
+     * - remove the @Ignore annotation from the test
+     * You should now be able to run the test against the docker container.
+     */
+    @Ignore
+    @Test
+    public void shouldCountByNino() {
```

Comment:
I think another test around this would be beneficial just to be sure that the count is counting what we think it is. Maybe put two different ninos in and check we count one only. Also checking that the date filter works seems prudent.

Reply:
will do